packages/framework/src/Model/Product/List/ProductListRepository.php

```diff
@@ -119,4 +119,13 @@ protected function getRepository(): EntityRepository
     {
         return $this->entityManager->getRepository(ProductList::class);
     }
+
+    /**
+     * @param string $uuid
+     * @return \Shopsys\FrameworkBundle\Model\Product\List\ProductList|null
+     */
+    public function findProductListByUuid(string $uuid): ?ProductList
+    {
+        return $this->getRepository()->findOneBy(['uuid' => $uuid]);
+    }
```

Comment:
this method is used only for checking the presence of the product list, but we're not interested in the data, right? 
What about using `$this->getRepository()->count()` to avoid unnecessary entity hydration?

Reply:
good idea! c81bd035ba